ninetofiver/views.py

```diff
@@ -28,6 +28,7 @@
 from rest_framework_swagger.renderers import SwaggerUIRenderer
 
 
+
```

Comment:
It's a better contrast between imports and functions imo.
I can remove it if you want to, but honestly don't see the point?